### PR TITLE
DCCM tool added - error with loading the lattice package

### PR DESCRIPTION
**Pull request by tsenapathi**

### CODE REVIEW

File: tools/bio3d/dccm.xml [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+           <when value="ligand"/>
+           <when value="nucleic"/>
+           <when value="elety">
+                <param name="elety"  type="text" value="CA" label="Atom Name"/>
+            </when>
+            <when value="resid">
+                <param name="resid"  type="text" value="BGLC" label="Resid"/>
+            </when>
+            <when value="segid">
+                <param name="segid"  type="text" value="SUBS" label="Segid"/>
```
Comment (bgruening) [inline tools/bio3d/dccm.xml]: Can you please add validators to all the text inputs and only allow letters?

File: tools/bio3d/macros.xml [hunk 2]
```diff
@@ -20,6 +19,7 @@
     </xml>
     <xml name="citations">
         <citations>
+            <citation type="doi">10.1093/bioinformatics/btz107</citation>
```
Comment (bgruening) [inline tools/bio3d/macros.xml]: this will now be shown on all tools, is that correct?

File: tools/bio3d/dccm.xml (hunk 1 above)
Comment (tsenapathi) [reply tools/bio3d/dccm.xml]: @bgruening this is a way to select different parts of the system (Different chains, different ligands). These are normally defined letters. You can also use numbers but normally in a pdb you only use letters.

File: tools/bio3d/macros.xml (hunk 2 above)
Comment (tsenapathi) [reply tools/bio3d/macros.xml]: Yes, We can add this right?

File: tools/bio3d/dccm.xml (hunk 1 above)
Comment (bgruening) [reply tools/bio3d/dccm.xml]: We can also use a validator for letters-and-numbers.

File: tools/bio3d/macros.xml (hunk 2 above)
Comment (bgruening) [reply tools/bio3d/macros.xml]: Sure, if this is correct.

**Comment by tsenapathi** [comment]
@chrisbarnettster For some reason, only the package specified first in the requirement section get installed

**Comment by tsenapathi** [comment]
@chrisbarnettster @bgruening Test Changed... tool worked! Now compare the size of the png output.

**Comment by bgruening** [comment]
merci! @tsenapathi can you add this tool to usegalaxy.eu as well please :)

**Comment by tsenapathi** [comment]
Thanks, @bgruening  I will :)

**Comment by tsenapathi** [comment]
@bgruening Done !!

**Pull request merged**